lib/chef/provisioning/transport/winrm.rb

```diff
@@ -36,8 +36,11 @@ def initialize(endpoint, type, options, global_config)
       def execute(command, execute_options = {})
         output = with_execute_timeout(execute_options) do
           session.set_timeout(execute_timeout(execute_options))
-          session.run_powershell_script(command) do |stdout, stderr|
-            stream_chunk(execute_options, stdout, stderr)
+          block = Proc.new { |stdout, stderr| stream_chunk(execute_options, stdout, stderr) }
+          if execute_options[:raw]
```

Comment:
these are really the only two modes that WinRMI supports, so I'm not sure that there'll be more. But I can totally make it a symbol.